### ISSUE TITLE
Odd migration behavior in 1.0.0 release

### ISSUE THREAD

**Issue by justinmayer** [bennylope/django-organizations#145]
Perhaps I am misunderstanding something, but I was confused by some odd migration behavior. Steps to reproduce:
```
virtualenv -p python3 foo
pip install Django django-extensions django-organizations
cd $VIRTUAL_ENV; django-admin startproject foo; cd foo
# add 'organizations' to INSTALLED_APPS
python manage.py migrate
python manage.py makemigrations
```
Expected result (which is what happens in `django-organizations` 0.9.3):
```
No changes detected
```
Actual result with `django-organizations` 1.0.0:
```
Migrations for 'organizations':
/virtualenvs/foo/lib/python3.6/site-packages/organizations/migrations/0003_auto_20180111_2242.py
    - Alter field slug on organization
```
It's not clear to me why the above migration is being auto-generated. I thought migrations were only auto-generated when model definitions differ from existing migrations, and I would have expected any needed migrations to be included with the app.

In short, I've never seen a Django app do this before, and thus it's not clear if the above is the intended behavior.

(Looking back, it's not clear to me why _existing_ migrations were modified in c1c88a7 — would it not have been better to create a new migration instead? cc: @AZtheAsian @nemesisdesign)

Again, perhaps I'm totally misunderstanding something. If indeed there's nothing amiss here, then I am merely seeking to be enlightened. 😇

**Comment by justinmayer** [comment]
Contents of the aforementioned auto-generated migration:
```
from django.db import migrations
import organizations.fields

class Migration(migrations.Migration):

    dependencies = [
        ('organizations', '0002_model_update'),
    ]

    operations = [
        migrations.AlterField(
            model_name='organization',
            name='slug',
            field=organizations.fields.SlugField(blank=True, editable=False, help_text='The name in all lowercase, suitable for URL identification', max_length=200, populate_from='name', unique=True),
        ),
    ]
```

**Comment by bennylope** [comment]
This is super annoying but unfortunately it's primarily a failure of the test process. There's a test that should have caught this but failed to do so.

Do you have this in production? Leaning toward just correcting the migration file itself rather than creating a new one, but that's not without its own issues.

**Comment by justinmayer** [comment]
Not yet. Was just about to put it into production when I noticed the issue. So I think correcting the migration file itself is an okay way to proceed this time.

**Comment by nemesisdesign** [comment]
Hey guys, definitely not the intended behaviour.
Would a new migration fix this?
I'll send a PR to improve the travis build.

**Comment by nemesisdesign** [comment]
Not sure how to do this with tox but here's the command to add:

    ./manage.py makemigrations --dry-run | grep "No changes detected"

**Comment by bennylope** [comment]
@nemesisdesign, thanks, but no PR needed for the tests, the last commit updated the migration test. It was present before but didn't catch this and has been updated to do so.

**Comment by nemesisdesign** [comment]
Seems like the test is working :-)
https://travis-ci.org/bennylope/django-organizations/jobs/327920706

**Comment by bennylope** [comment]
Waiting to have some time to carve out to verify what's going on in my own environment and why I'm generating different migrations with tox vs. the local dev virtualenv.

**Comment by jdp** [comment]
This is happening for me as well. Including my requirements.txt to potentially aid in debugging:

```
Django==2.0.2
django-bootstrap4==0.0.5
django-braces==1.12.0
django-extensions==1.9.9
django-organizations==1.0.0
flake8==3.5.0
mccabe==0.6.1
pycodestyle==2.3.1
pyflakes==1.6.0
pytz==2018.3
six==1.11.0
typing==3.6.4
```

**Comment by nemesisdesign** [comment]
@bennylope thx, seems can be closedd because fixed by 79bd899?

**Comment by justinmayer** [comment]
I'm confused. I've looked at 79bd899 and still can't tell whether the `django-extensions` slug field is supported and/or the default, as I would hope it would continue to be. The README says the `django-extensions` slug field is the default, but then it's commented out in a few places in the aforementioned commit.
